src/task_queue/database_tasks/menu_backfill.py

```diff
@@ -0,0 +1,40 @@
+import logging
+
+from api.utils.api_utils import get_viable_menu_items
+from database.models import Community, Menu
+
```

Comment:
I see this is a task, but have no idea what its purpose is.  @abdullai-t could you write a comment at the top of any routine like this and say what it is for, whether it needs to be run once or more than once?

Reply:
Ok. Will do that